src/widgets/modalChildren/BreachDisplay.js

```diff
@@ -19,12 +19,16 @@ import { AfterInteractions } from '../AfterInteractions';
 
 const Breach = ({ breach, getInfoColumns }) => {
   const { temperatureLogs } = breach;
+  const lineData = temperatureLogs.map(log => {
+    const { timestamp, temperature } = log;
+    return { timestamp, temperature };
+  });
```

Comment:
Hm 🤔 confused why this is needed

Reply:
well spotted!
uh, it's not. I went wandering down that path thinking that I might have to re-sort the data. didn't in the end. I became convinced that it made the chart slightly faster to load, so kept it in 🤷‍♂️